app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,4 +1,11 @@
 class Admin::DashboardController < Admin::BaseController
+  before_action :check_authorize
+
   def index
   end
+
+  private
```

Comment:
__[RuboCop]__

Layout/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.


<!-- Comment by [SideCI](https://sideci.com) -->